src/templates/file.html

```diff
@@ -0,0 +1,38 @@
+{% import "macros.html" as macros %}
+{% extends "base.html" %}
```

Comment:
nit: 
![image](https://user-images.githubusercontent.com/5641725/101044503-b04eb900-357f-11eb-9876-5cb0d5fb692b.png)
It'd be nice to have no warning: see https://validator.w3.org/ to check yourself

Reply:
It had been addressed in the updated set.